### PR TITLE
Improve Subscription Liveness Logging

### PR DESCRIPTION
**Pull request by mrjerryjohns**
Improve the logging of subscription establishment and liveness printouts
to contain more information about the subscription and the server being
subscribed to.

### CODE REVIEW

File: src/app/ReadClient.cpp [hunk 1]
```diff
@@ -693,7 +693,10 @@ CHIP_ERROR ReadClient::RefreshLivenessCheckTimer()
     System::Clock::Timeout timeout =
         System::Clock::Seconds16(mMaxIntervalCeilingSeconds) + mpExchangeCtx->GetSessionHandle()->GetAckTimeout();
     // EFR32/MBED/INFINION/K32W's chrono count return long unsinged, but other platform returns unsigned
-    ChipLogProgress(DataManagement, "Refresh LivenessCheckTime with %lu milliseconds", static_cast<long unsigned>(timeout.count()));
+    ChipLogProgress(DataManagement,
+                    "Refresh LivenessCheckTime for %lu milliseconds with subscription id = 0x%" PRIx64 ", Peer = F:%" PRIx8
```
Comment (trombadore) [inline src/app/ReadClient.cpp:697]: why F and N, why not FabricID and NodeID? Seems more clear

**Review by trombadore-google — APPROVED** [review]

File: src/app/ReadClient.cpp (hunk 1 above)
Comment (mrjerryjohns) [reply src/app/ReadClient.cpp:697]: I'm trying to establish a compact logging format if you will that we can use in plenty of the other places where we log similar bits of information. In doing so, was trying to reduce the wordiness.

Comment (bzbarsky-apple) [inline src/app/ReadClient.cpp:697]: ChipLogFormatX64, not PRIx64.  And PRIx8 is not allowed; note it's failing a lint.  Use "%x" for that bit.

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

**Review by trombadore-google — CHANGES_REQUESTED** [review]

File: src/app/ReadClient.cpp (hunk 1 above)
Comment (mrjerryjohns) [reply src/app/ReadClient.cpp]: I've just gone with `Peer = 01:0000000000000002`. I think that should be fairly self explanatory.

Comment (mrjerryjohns) [reply src/app/ReadClient.cpp]: In my experience, having brevity helps when scanning logs for debugging, which I've done many times in the past with Weave, which had a similar system. That brevity helped when scanning logs for figuring out whats happening.

**Review by trombadore-google — CHANGES_REQUESTED** [review]

**Review by trombadore-google — APPROVED** [review]
LGTM

**Review by bzbarsky-apple — APPROVED** [review]

**Review by trombadore-google — APPROVED** [review]

**Review by yunhanw-google — APPROVED** [review]

**Pull request merged**